configure.ml

```diff
@@ -109,7 +109,7 @@ let configure bindir sbindir libexecdir scriptsdir etcdir mandir =
   let configmllines =
     [ "(* Warning - this file is autogenerated by the configure script *)";
       "(* Do not edit *)";
-      Printf.sprintf "#let xen45 = %b" xen45
+      Printf.sprintf "#define xen45 %d" (if xen45 then 1 else 0) (* cppo is a bit broken *)
```

Comment:
@lindig No, it seems that the `#let` syntax of [optcomp](https://github.com/diml/optcomp) (which this PR replaces with cppo), isn't supported by [cppo](https://github.com/mjambon/cppo). 

From the comment it seems that @jonludlam , who is the original author of these commits, encountered a `cppo` bug, and I think probably this forced him to not use `#ifdef`.